opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py

```diff
@@ -319,9 +321,24 @@ def __init__(
 
     @staticmethod
     def _get_attributes(record: logging.LogRecord) -> Attributes:
-        return {
+        attributes = {
             k: v for k, v in vars(record).items() if k not in _RESERVED_ATTRS
         }
+        if record.exc_info:
+            exctype, value, tb = record.exc_info
+            if exctype is not None:
+                exc_type = exctype.__name__
+            if value is not None:
+                message = value.args[0]
```

Comment:
There is an edge case here. If the exception is raised like this:

``` python
try:
    raise Exception()  # No arguments in the parentheses
except:
    logger.exception(...)
```
then `value.args` will be empty.

Reply:
Is that not fine? Message will just be empty.